# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/moz.build
# Commit: 63603267402e
# Full Hash: 63603267402ee93d6551ef60c35191d99aaee06b
# Author: Zach Hoffman <zach@zrhoffman.net>
# Date: 2023-12-18 17:17:57
# Regressor Bug: 1846516
# File Overlap Count: 3
# Description:
#   Bug 1846516 - [css-properties-values-api] Introduce AnimatedPropertyID/AnimatedPropertIDSet. r=firefox-animation-reviewers,firefox-style-system-reviewers,layout-reviewers,emilio,hiro
#   
#   This will make possible to animate custom properties on Gecko side. For now, the
#   animation code keeps only dealing with nsCSSPropertyID, so behavior is unchanged.
#   
# ==============================================================================

diff -r 753bc39f59ff -r 63603267402e layout/style/moz.build
--- a/layout/style/moz.build	Mon Dec 18 09:24:07 2023 +0000
+++ b/layout/style/moz.build	Mon Dec 18 09:24:08 2023 +0000
@@ -22,6 +22,9 @@
 with Files("AnimationCollection.*"):
     BUG_COMPONENT = ("Core", "CSS Transitions and Animations")
 
+with Files("AnimatedPropertyID*"):
+    BUG_COMPONENT = ("Core", "CSS Transitions and Animations")
+
 with Files("AnimationCommon.*"):
     BUG_COMPONENT = ("Core", "CSS Transitions and Animations")
 
@@ -67,6 +70,8 @@
 
 EXPORTS.mozilla += [
     "!ServoCSSPropList.h",
+    "AnimatedPropertyID.h",
+    "AnimatedPropertyIDSet.h",
     "AnimationCollection.h",
     "AttributeStyles.h",
     "CachedInheritingStyles.h",